### PR TITLE
Add k8s-annotations-exporter to osc clusters

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/apps/issues/1819

### CODE REVIEW

**Comment by HumairAK** [comment]
@redmikhail fy this will add a metrics exporter to these clusters, it allows us to query ocp `namespace` `annotations` within the `namespace` objects using prometheus/grafana. The highest perms it gets is cluster reader.

File: argocd/overlays/moc-infra/applications/envs/osc/osc-cl1/operate-first/k8s-annotations-exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: k8s-annotations-exporter
+spec:
+  destination:
+    name: osc-cl1
+    namespace: opf-monitoring
+  project: operate-first
```
[the remark this answers is not in the document]
Comment (4n4nd) [reply argocd/overlays/moc-infra/applications/envs/osc/osc-cl1/operate-first/k8s-annotations-exporter.yaml]: updated :+1:

**Comment by HumairAK** [comment]
@redmikhail no problem!

**Comment by 4n4nd** [comment]
@redmikhail just to make it clear, this won't be exporting metrics outside the OSC clusters. This will just add new metrics for the openshift user workload monitoring prometheus to scrape.

**Comment by HumairAK** [comment]
>  this won't be exporting metrics outside the OSC clusters. 

This will be a separate pr, and is the current intended goal

**Comment by redmikhail** [comment]
Thank you , @4n4nd and @HumairAK !

**Pull request merged**